README.md

```diff
@@ -328,6 +329,13 @@ function shouldRenderSuggestions(value) {
 }
 ```
 
+<a name="alwaysRenderSuggestionsProp"></a>
+#### alwaysRenderSuggestions (optional)
+
+By default, suggestions are rendered only if the input field is under focus.
+
+If you'd like to _always_ display suggestions, set `alwaysRenderSuggestions={true}`.
```

Comment:
I was wondering if having the two documentation sections next to one another would be enough. Doesn't hurt to make this info more explicit.